### DISCUSSION TITLE
Load Save state automatically only when using game switcher?

### DISCUSSION THREAD

**Discussion by erfanCA01** [OnionUI/Onion#1407] [Ideas]
Hello, I love the game switcher with auto load state feature, but when i am selecting a rom from the favorites or main rom list, i don't like that the game auto loads the state when opening. I'd like to watch through the intro when i can, I know i can turn off auto load state but it also seems to turn off the auto load state in game switcher too, it feels a bit disjointed when running game switcher and it has the last screenshot of the game but doesnt auto load when using it.

I am not sure if i can already do the "Load Save state automatically only when using game switcher?" thing but if not i think this is a great customization option!

**Comment by Aemiii91** [comment]
I agree 💯

**Reply by erfanCA01** [reply]
OnionOS already has the "reset game" option from GLO menu, **so if the reset script would trigger everytime the game is launched outside of gameswitcher**, then it would do exactly as i described in my initial post 😁, like selecting the game from favourites or game list, just like stock OS, but when you get into it from gameswitcher , it autoloads as usual!

not sure if i can already do it now or a dev needs to implement it? maybe we can have a tweaks like app for gameswitcher too? to customize these options, another game switcher option i would like is displaying the box art of the game instead of last played screenshot
